Partial-CI-Tests-Structure.md

```diff
@@ -0,0 +1,17 @@
+## Introduction
+
+If your PR fails due to the `check_ci_test_suites_to_run` workflow, please look at the error log and depending on the error, do the following:
+
+* If you see an error about the root files config, then see the [root files config section](#root-files-config) below.
+
+## Overview of Partial CI Tests Structure
+
+For non-python files, we run partial ci tests depending on the tests that a specific file affects. The process of determining which tests to run goes like this:
```

Comment:
Maybe add a sentence after the first one to explain the benefits of doing this.

Reply:
Done.